packages/api-fetch/src/middlewares/nonce.js

```diff
@@ -11,7 +11,7 @@ const createNonceMiddleware = ( nonce ) => ( options, next ) => {
 	 * Configure heartbeat to refresh the wp-api nonce, keeping the editor
 	 * authorization intact.
 	 */
-	jQuery( document ).on( 'heartbeat-tick', ( event, response ) => {
+	addAction( 'heartbeat.tick', 'api-fetch/createNonceMiddleware', ( response ) => {
```

Comment:
Can we rename `api-fetch/createNonceMiddleware` to `core/api-fetch/create-nonce-middleware`? It would make it more similar to other hooks.

Reply:
yeah, it makes sense. renamed!